packages/forms-web-app/src/validators/appellant-submission/supporting-documents.js

```diff
@@ -0,0 +1,7 @@
+const rules = () => {
```

Comment:
there are validation rules, but they are covered under the extend form web app ticket